### ISSUE TITLE
support for business notebooks

### ISSUE THREAD

**Issue by chinarut** [VitaliyRodnenko/geeknote#214]
thanks for leading the pack on a CLI for Evernote!

when I list notebooks, I only see my personal notebooks, not any of my business notebooks.

please support business notebooks as this is probably where automation is most valuable :)

**Comment by tha-dok** [comment]
I second this notion.

**Comment by myw** [comment]
yes, please! the API has changed recently and is apparently less awful to work with.

**Comment by ygordev** [comment]
Any news about business netbooks support?
